src/services/goToDefinition.ts

```diff
@@ -510,4 +517,22 @@ namespace ts.GoToDefinition {
                 return false;
         }
     }
+
+    function createDefinitionInfoFromSwitchStatement(statement: SwitchStatement): DefinitionInfo {

```

Comment:
Maybe we just need a `createDefinitionInfoFromStatement` since we don't need to make this specific to any keyword.

Reply:
```